internal/pkg/selfupdate/update.go

```diff
@@ -37,11 +33,18 @@ func UpgradeInPlace(ctx context.Context, curSV semver.Version, baseSiteURL strin
 
 	command := updateCommand(baseSiteURL)
 
+	tmp, err := os.MkdirTemp("", "*")
+	if err != nil {
+		return fmt.Errorf("making temporary directory in order to install new binary: %v", err)
+	}
+	defer os.RemoveAll(tmp)
+
 	cmd := exec.Command(shellToUse, switchToUse, command)
 	cmd.Stdout = stdout
 	cmd.Stderr = stderr
 	cmd.Stdin = stdin
 	cmd.Env = append(cmd.Env, "INSIDE_HUMANLOG_SELF_UPDATE=true")
+	cmd.Env = append(cmd.Env, fmt.Sprintf("HUMANLOG_INSTALL=%s", tmp))
```

Comment:
```suggestion
	cmd.Env = append(cmd.Env, "HUMANLOG_INSTALL=" + tmp)
```
don't use `fmt.Sprintf` when you're just doing string concatenation :P

string concat is a language built-in, and `fmt.Sprintf` is a mini-interpreter. it doesn't matter here, but it's just not a good practice to `Sprintf` unless needed (or because it makes readability much better).